scripts/keeper.py

```diff
@@ -0,0 +1,146 @@
+
+import asyncio
+import json
+import time
+import sys
+import os 
+import argparse
+import logging
+from typing import List
+
+from starknet_py.contract import Contract
+from starknet_py.net import AccountClient, KeyPair
+from starknet_py.net.account.account import Account
+from starknet_py.net.gateway_client import GatewayClient
+from starknet_py.net.models.chains import StarknetChainId
+from starknet_py.net.client_models import TransactionStatus
+from starknet_py.transaction_exceptions import TransactionRejectedError
+
+MAX_FEE=int(1e16)
+SUPPORTED_NETWORKS = ['testnet', 'mainnet']
+
+def parse_envs() -> List[str]:
+    PRIVATE_KEY = os.getenv('PRIVATE_KEY')
+    PUBLIC_KEY = os.getenv('PUBLIC_KEY')
+
+    if PRIVATE_KEY == None:
+        raise ValueError("Missing PRIVATE_KEY ENV")
+
+    if PUBLIC_KEY == None:
+        raise ValueError("Missing PUBLIC_KEY ENV")
+
+    return [PRIVATE_KEY,PUBLIC_KEY]
+
+def setup_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser(
+                        prog = 'Starknet Keeper bot',
+                        description = 'Periodically calls predefined function on Starknet',
+            )
+
+    parser.add_argument(
+        '--net', '-n', type = str
+    )
+    parser.add_argument(
+        '--wallet_address', '-wa', type = str
+    )
+    parser.add_argument(
+        '--contract_address', type = str
+    )
+    parser.add_argument(
+        '--abi_path', type=str
+    )
+    parser.add_argument(
+        '--function_name', '-f', type = str
+    )
+    parser.add_argument(
+        '--function_arguments', '-fa', action='append', default = []
+    )
+    return parser
+
+def get_abi(abi_path: str) -> List:
+    with open(abi_path, 'r') as f:
+        abi = json.load(f)
+
+    return abi
```

Comment:
Maybe sprinkle in a few try catches? 😄 
```suggestion
def get_abi(abi_path: str) -> List:
    try:
        f = open(abi_path, 'r')
    except:
        raise ValueError("Invalid ABI path")
    with f:
        try:
            abi = json.load(f)
        except:
            raise ValueError("Invalid ABI file")

    return abi
```